oximeter/oximeter/src/lib.rs

```diff
@@ -185,14 +185,15 @@
 //! `Producer`s may be registered with the same `ProducerServer`, each with potentially different
 //! sampling intervals.
 
-pub use oximeter_impl::*;
+pub use oximeter_macro_impl::{Metric, Target};
 pub use oximeter_timeseries_macro::use_timeseries;
+pub use oximeter_types::*;
```

Comment:
Ha, I was surprised that we don't need to reexport `oximeter-schema` here! But it is only for tests actually, so it seems OK the way it is!